libs/sdk-ui-filters/src/AttributeFilter/hooks/useResolveDependentDateFiltersInput.ts

```diff
@@ -25,8 +25,12 @@ export const useResolveDependentDateFiltersInput = (
                 const { dataSet, granularity, from, to } = dependentDateFilter.dateFilter;
 
                 if (isRelativeDashboardDateFilter(dependentDateFilter)) {
-                    // Ignore date filters set as "All time"
-                    if (granularity === "GDC.time.date") {
+                    // Ignore only date filters set as "All time"
+                    if (
+                        granularity === "GDC.time.date" &&
+                        // eslint-disable-next-line no-prototype-builtins
+                        !dependentDateFilter.dateFilter.hasOwnProperty("from")
```

Comment:
Can this be rewritten to type guard which would eliminate the eslint disable statement?

Reply:
Added typeguard.